packages/theme/src/theme/_forms.scss

```diff
@@ -333,6 +333,11 @@ form {
 			pointer-events: none;
 		}
 
+		+ .form-control-title {
+			position: absolute;
+			top: -10px;
+		}
+
```

Comment:
Instead of overriding the style, you can override the `labelAfter` prop that determine if it is put before or after the input. If it is after, the floating style applies.
If you have a hint, you force the label to be before